docs/plugins/spectate.rst

```diff
@@ -103,10 +103,22 @@ Settings
     Toggle whether to prefer following (non-siege) units that have newly
     arrived on the map.
 
+``tooltip-follow`` (default: enabled)
+    If the ``spectate.tooltip`` overlay is enabled, toggle whether to show the
+    tooltip.
+
+``tooltip-follow-blink-milliseconds`` (default: 3000)
```

Comment:
we'll have to get feedback on this in the beta. It might make sense to default to always on for graphics mode but default to a slow blink for ascii. Other tools that display things over the map do that, like the warm/damp overlay and the gui/quickfort preview shadow.